templates/application/migrator/lightblue-migrator-daemon.sh.erb

```diff
@@ -45,7 +45,7 @@ do_exec()
   
   if @jvmOptions
     @jvmOptions.each do |key, value|
-      jsvcCommand << "-X#{key}=#{value}"
+      jsvcCommand << "-X#{key}#{value}"
```

Comment:
Noticed there isn't any space from one option to another , could you create a test to see if it work fine please? (maybe using Xms and Xmx, or can be any other parameter). Also, the jvmOptions might be used for some parameters that need '=' sign and other things (there are a lot of JVM parameters)